client/src/components/Verejne/entityHelpers.js

```diff
@@ -0,0 +1,16 @@
+// Entity which is a number is individual.
+// District starts with letter
+export const isIndividual = (eid) => {
+  return !isNaN(parseFloat(eid)) && isFinite(eid)
```

Comment:
this seems like a weird check for what you're saying you need to check ... why the finite test ?